tensorflow/stream_executor/rocm/rocm_blas.h

```diff
@@ -110,6 +110,89 @@ class ROCMBlas : public blas::BlasSupport {
                               /*err_on_failure=*/false, args...);
   }
 
+
+
+  // simply reorders the arguments
+  // Batched GEMM interface
+  template <typename FuncT, typename T>
+  bool DoBlasCall(const T* a, const T* b, T* c,
+      const T* alpha_ptr, const T* beta_ptr, 
+      FuncT rocblas_func, Stream* stream,
+      rocblas_operation transa, rocblas_operation transb, uint64 m, uint64 n, uint64 k,
+      int lda, int ldb, int ldc)
+  {
+    return ROCMBlas::DoBlasInternal(rocblas_func, stream, true,
+                             transa, transb,
+                             m, n, k, alpha_ptr, a, lda,
+                             b, ldb, 
+                             beta_ptr, c, ldc);
+  }
```

Comment:
What is the motivation behind those one statement function wrappers? It looks like `DoBlasInternal` itself is a straightforward redirect function too.

Reply:
For code reuse. This way I can use the same DoBlasComplex for GEMM and batched GEMM. 

I suppose DoBlasCall could call rocblas_func directly to save one abstraction layer